src/Catalog/Helpers/Db2CatalogProjection.cs

```diff
@@ -34,6 +34,7 @@ public FeedPackageDetails ReadFeedPackageDetailsFromDataReader(DbDataReader data
 
             var packageId = dataReader[Db2CatalogProjectionColumnNames.PackageId].ToString();
             var normalizedPackageVersion = dataReader[Db2CatalogProjectionColumnNames.NormalizedVersion].ToString();
+            var fullPackageVersion = dataReader[Db2CatalogProjectionColumnNames.FullVersion].ToString();
```

Comment:
Unfortunately, monitoring blobs are stored based on a package's "full version" and not their "normalized version". This means that I needed to update the package fetching code used by `Db2Catalog` to grab this information as well. I felt like there isn't very much risk in this but I understand there may be some concerns.

Reply:
![This is fine](https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif)